gooddata-pandas/gooddata_pandas/result_convertor.py

```diff
@@ -124,6 +124,28 @@ def result(self) -> _DataWithHeaders:
         )
 
 
+@define
+class DataFrameMetadata:
+    # row line location where total header is located per index header column
+    # example:
+    # Category  | Country | Budget
+    # Car       | Arizona |    100
+    #           | Texas   |     50
+    #           | SUM     |    150
+    # Train     | Arizona |    200    =>    [[7],[3,6]]
+    #           | Texas   |    100
+    #           | AVG     |    150
+    # SUM       |         |    450
+    row_totals_indexes: List[List[int]]
+
+    @classmethod
+    def from_data(cls, headers: Tuple[_DataHeaders, Optional[_DataHeaders]]) -> "DataFrameMetadata":
```

Comment:
I noticed that I am unsure about headers being a tuple, and I am missing a context of what is located in the tuple then. Did you consider using namedtuple or data class to make it more readable with proper variables explaining the context?

Reply:
It's: `Tuple[<row header values = axis y>, <column header values = axis x>]`

Also stated in a more vague sentence in 
```
@frozen
class _DataWithHeaders:
    data: List[_DataArray]
    """extracted data; either array of values for one-dimensional result or array of arrays of values"""

    data_headers: Tuple[_DataHeaders, Optional[_DataHeaders]]
=>     """per-dimension headers for the data"""
```

Reply:
As it's not related to this ticket I would rather create a follow-up for better readibility if necessary.